daffodil-cli/src/it/scala/org/apache/daffodil/performance/TestCLIPerformance.scala

```diff
@@ -180,7 +180,7 @@ class TestCLIPerformance {
       if (exitCode == 0)
         fail("Tests were successful when they were expected to fail. Exit code: " + exitCode)
 
-      Util.expectExitCode(ExitCode.PerformanceTestError, shell)
+      Util.expectExitCode(ExitCode.Success, shell)
```

Comment:
It looks like this tests is expecting things to fail, and so shouldn't be ExitCode.Success. If this test is passing and exiting with Zero I think there is a bug somewhere.